zulipterminal/cli/run.py

```diff
@@ -439,6 +440,10 @@ def main(options: Optional[List[str]] = None) -> None:
         if args.notify:
             zterm["notify"] = (args.notify, "on command line")
 
+        if PLATFORM:
+            print("Detected Platform: " + PLATFORM)
+        else:
+            print("Running on undetected platform.")
```

Comment:
> Is this intended to be for unsupported platforms, or something else?
> 
> Please make sure you're aware what values this can take :)
> 
> You're not testing this result, in any case.

I'll look into that; I'm still not exactly sure how to check for unsupported platforms, though.